schedule.html

```diff
@@ -44,152 +45,161 @@
-
-                    <tr>
-                        <td>Closing remarks, winner announced</td>
-                        <td>2:40pm - 3:10pm</td>
-                    </tr>
-
-                </tbody>
-            </table>
-<!--        </div>-->
-    </div>
-    <br>
-    <br>
+<br>
+<div class="HackYU-Schedule">
+    <!--        <div class="schedule-column">-->
+    <table class="table table-hover">
+        <thead>
+        <tr>
+            <th scope="col">Kick-Off</th>
+            <th scope="col">9:30pm - 10:20pm</th>
+        </tr>
+        </thead>
+
+        <tbody>
+
+        <tr>
+            <td>Sign-in</td>
+            <td>9:30pm - 10:00 pm</td>
+        </tr>
+
+        <tr>
+            <td>Opening remarks</td>
+            <td>10:00pm - 10:20pm</td>
+        </tr>
+
+        </tbody>
+
+        <thead>
+        <tr>
+            <th scope="col">Coding</th>
+            <th scope="col">10:30pm - 1:30pm</th>
+        </tr>
+        </thead>
+
+        <tbody>
+
+        <tr>
+            <td>Workshop #1: <br/>
+                <div class="workshop-details">Noam Annenberg,</div>
+                <div class="workshop-details">"Building a Full Stack Web App"</div></td>
+            <td>10:45pm - 11:15pm</td>
+        </tr>
+
+        <tr>
+            <td>Workshop #2: <br/>
+                <div class="workshop-details"> Yosef Sklar,</div>
+                <div class="workshop-details">"From HWs to Hackathons:
+                    How what you’re learning in school translates to web applications"</div></td>
+            <td>11:30pm - 12:00am</td>
+        </tr>
+
+        <tr>
+            <td>Midnight dinner (meat)</td>
+            <td>12:05am - 12:40am</td>
+        </tr>
+
+        <tr>
+            <td>Game</td>
+            <td>12:40am - 1:10am</td>
+        </tr>
+
+        <tr>
+            <td>Just coding!</td>
+            <td>1:10am - 8:00am</td>
+        </tr>
+
+        <tr>
+            <td>Shacharit</td>
+            <td>8:00am - 8:45am</td>
+        </tr>
+
+        <tr>
+            <td>Breakfast (dairy)</td>
+            <td>9:00am - 9:30am</td>
+        </tr>
+
+        <!--                </tbody>-->
+        <!--            </table>-->
+        <!--        </div>-->
+
+        <!--        <div class="schedule-column">-->
+        <!--            <table class="table table-hover">-->
+        <!--                <thead>-->
+        <!--                    <tr>-->
+        <!--                        <th scope="col">Coding</th>-->
+        <!--                        <th scope="col">10:30pm - 1:30pm</th>-->
+        <!--                    </tr>-->
+        <!--                </thead>-->
+
+        <!--                <tbody>-->
+
+        <tr>
+            <td>Workshop #3: <br/>
+                <div class="workshop-details">Aryeh Klein, </div>
+                <div class="workshop-details">"Leave your computer behind: A basic introduction to AWS and how to
+                    use it"</div>
```

Comment:
```suggestion
                <div class="workshop-details">"A Basic Introduction to AWS & How to Use It"</div>
```
This is a shorter title of the workshop that I found on the google doc